api/dao/containerutil.py

```diff
@@ -192,3 +193,18 @@ def create_containerreference_from_dictionary(d):
 
 def create_containerreference_from_filereference(fr):
     return ContainerReference.from_filereference(fr)
+
+
+def pluralize(cont_name):
```

Comment:
Like I've mentioned before, these are great. A much better implementation than the makeshift functions we have scattered around the products.